lib/helpers/todosHelper.js

```diff
@@ -83,5 +83,11 @@ module.exports = {
 		return items.every(function (item) {
 			return item.isCompleted;
 		});
+	},
+
+	enumerate: function (items) {
+		items.forEach(function (item, index) {
+			item.setId(index);
```

Comment:
It's not good to modify id. It's better to use a sequence to generate id in `append`